python/test/unit/spacewalk/satellite_tools/test_yum_src.py

```diff
@@ -54,7 +55,7 @@ def _make_dummy_cs(self):
 
         yum_src.get_proxy = Mock(return_value=(None, None, None))
 
-        cs = yum_src.ContentSource("http://example.com", "test_repo", org='')
+        cs = yum_src.ContentSource("http://example.com/fake_path/", "test_repo", org='')
```

Comment:
Just something I didn't quite understand, why is this needed?

Reply:
I just noticed there was some failures during test execution when using a URL like that, with no real path, as `ContentSource` URL.

There is some code around in reposync and "yum_src" that is apparentely expecting that there is always certain relative path. For example: https://github.com/uyuni-project/uyuni/blame/master/python/spacewalk/satellite_tools/reposync.py#L567